.github/workflows/ci.yml

```diff
@@ -104,8 +104,11 @@ jobs:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           native-image-job-reports: "true"
           cache: "maven"
-      - name: "Native"
-        run: mvn -Pnative -DskipTests package -pl core -am && util/test-native.sh
+      - name: "Native Build"
+        run: mvn -Pnative -DskipTests package -pl core -am
+      - name: "Native Test"
+        if: ${{ matrix.os != 'windows-latest' }}
+        run: util/test-native.sh
```

Comment:
To check my understanding, we're skipping this because the test script isn't compatible with Windows, but we expect the native-image output still will be, right?

Maybe add a comment?

Reply:
> To check my understanding, we're skipping this because the test script isn't compatible with Windows, but we expect the native-image output still will be, right?

Yup, that Bash script for testing won't work on Windows. Someone could write a `*.bat` or `*.ps1` equivalent later, but I'm not up for that - let's get this in as-is (from my side).

> Maybe add a comment?

Done!

Reply:
Thanks!